babel.config.js

```diff
@@ -3,20 +3,56 @@ const config = {
 		[
 			'@babel/env',
 			{
-				corejs: 2,
-				modules: 'commonjs',
 				useBuiltIns: 'entry',
+				corejs: 2,
+				// Exclude transforms that make all code slower, see https://github.com/facebook/create-react-app/pull/5278
+				exclude: [ 'transform-typeof-symbol' ],
 			},
 		],
 		'@babel/react',
 	],
 	plugins: [
-		'@babel/proposal-class-properties',
-		'@babel/transform-runtime',
-		'@babel/transform-member-expression-literals',
-		'@babel/transform-property-literals',
-		'@babel/transform-reserved-words',
```

Comment:
>These specific transforms are removed:
>
>    @babel/transform-member-expression-literals
>    @babel/transform-property-literals
>    @babel/transform-reserved-words
>
>It's unclear why they may have been necessary historically.

At least `@babel/transform-reserved-words` was added in the [previous bigger Babel upgrade](https://github.com/Automattic/jetpack/pull/10810/files#diff-fc10a6682269c1fefc2abe05d1500b8aR18) not long ago. Others seemed to be there from before already.

cc @jsnajdr @brbrr for some insight into those dependencies.

Reply:
> The `transform-member-expression-literals` plugins are needed for ES3 compatibility with old IEs that Jetpack still keeps (?). `test.catch` is syntax error in ES3, because `catch` is reserved word. It's transformed to `test['catch']`.

[via](https://github.com/Automattic/jetpack/pull/10810#discussion_r268185649)

Reply:
These transforms are need to transpile some constructs that are legal in ES5, but are not in ES3. Like using reserved keywords as property names:
```js
a.class /* is illegal and is changed to */ a[ 'class' ]
```
They were there before upgrade to Babel 7. Just the names of the transforms were different. They are needed for compatibility with IE 9 or IE 10. I've been told that in Dec 2018 by someone from Jetpack Crew. But I don't remember who, can't find the message now.

I don't know if ancient IE compatibility is still needed or if it even works. Maybe there are further issues beyond the reserved-keywords-as-properties? I never tested that.

Reply:
> don't know if ancient IE compatibility is still needed or if it even works.

We now support from IE11 upwards only 👍 

https://github.com/Automattic/jetpack/blob/93cfaa63126a31c25e596c73ab78ded6d66d808a/docs/coding-guidelines.md#versions-supported

Reply:
Woohoo! Means we can :hocho: that transform I guess :grimacing: